src/main/rx/crsf.c

```diff
@@ -141,20 +149,21 @@ STATIC_UNIT_TESTED void crsfDataReceive(uint16_t c)
         crsfFrameDone = crsfFramePosition < fullFrameLength ? false : true;
         if (crsfFrameDone) {
             crsfFramePosition = 0;
+#if defined(USE_MSP_OVER_TELEMETRY)
+            if (crsfFrame.frame.type == CRSF_FRAMETYPE_MSP_REQ || crsfFrame.frame.type == CRSF_FRAMETYPE_MSP_WRITE) {
+                const uint8_t crc = crsfFrameCRC();
+                if (crc == crsfFrame.bytes[fullFrameLength-1]) {
+                    uint8_t *frameStart = (uint8_t *)&crsfFrame.frame.payload + CRSF_FRAME_ORIGIN_DEST_SIZE;
+                    if (bufferCrsfMspFrame(frameStart, CRSF_FRAME_RX_MSP_PAYLOAD_SIZE)) {
```

Comment:
Looks like this does not include the CRC byte.

Reply:
`fullFrameLength` is 1-indexed while `crsfFrame.bytes[]` is 0-indexed.  The `-1` lines up the indexing for the CRSF CRC check.  This byte is no longer needed once it has been checked.  The MSP CRC is contained within the MSP payload which could be any one of the MSP bytes in `CRSF_FRAME_RX_MSP_PAYLOAD_SIZE`.  I have checked this deeply with a debugger and the CRCs are validated for both CRSF and MSP.   If they weren't, the frames would not be processed.

Reply:
I could rename this constant to `CRSF_FRAME_RX_MSP_FRAME_SIZE` if that will make things clearer to others.